docs/Using-Inbox-Service.md

```diff
@@ -0,0 +1,151 @@
+# Using Inbox Service
+
+<!-- begin remove -->
+- [Introduction](#introduction)
+- [Creating an Instance](#creating-an-instance)
+- [Inbox Service Usage](#inbox-service-usage)
+  - [Get Number of Unread Messages](#get-number-of-unread-messages)
+  - [Get List of Messages](#get-list-of-messages)
+  - [Get Message Detail](#get-message-detail)
+  - [Set Message as Read](#set-message-as-read)
+- [Error handling](#error-handling)
+<!-- end -->
+
+## Introduction
```

Comment:
No need to include the "Introduction" heading in the introduction. The developer portal will generate this in navigation automatically, and in the generated document, it does not look good to have H1 immediately followed by H2.